apps/stellar-contracts/tests/cli/cleanup.sh

```
@@ -0,0 +1,314 @@
+REVIEW_CONTRACT_ID=$(get_contract_address "review_contract")
+
+cat >> "$SUMMARY_FILE" << EOF
+    "booking": "$BOOKING_CONTRACT_ID",
+    "property_listing": "$PROPERTY_CONTRACT_ID",
+    "review_contract": "$REVIEW_CONTRACT_ID"
+  },
+  "test_reports": [
+EOF
+
+# Add each test report to summary
+for i in "${!TEST_REPORTS[@]}"; do
```

Shellcheck (0.11.0)

[error] Couldn't parse this for loop. Fix to allow more checks.

(SC1073)